### PR TITLE
Test factories

### PR DESCRIPTION
**Pull request by Rafalkufel**
__Description:__

Test for UserFactory and OrganizationFactory

### CODE REVIEW

File: backend/apps/volontulo/factories/factories.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        'Poradnia': 'zenski',
+        'Szpital': 'meski',
+        'Ogród': 'meski',
+        'Koło': 'nijaki',
+        'Obwód': 'meski'
+        }
+    predicate2_dict = {
+        'meski': [
+            'Organizacyjny', 'Rejestrowy', 'Egzekutywny', 'Wspierający',
+            'Tranakcyjny', 'Związkowy', 'Zbiorczy'],
```
Comment (w1stler) [inline backend/apps/volontulo/factories/factories.py]: Transakcyjny - check spelling ;-)

File: backend/apps/volontulo/factories/factories.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        'zenski': [
+            'Organizacyjna', 'Rejestrowa', 'Egzekutywna', 'Wspierająca',
+            'Tranakcyjna', 'Związkowa', 'Zbiorcza'],
```
Comment (w1stler) [inline backend/apps/volontulo/factories/factories.py]: ditto

File: backend/apps/volontulo/factories/factories.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        'nijaki': [
+            'Organizacyjne', 'Rejestrowe', 'Egzekutywne', 'Wspierające',
+            'Tranakcyjne', 'Związkowe', 'Zbiorcze']
```
Comment (w1stler) [inline backend/apps/volontulo/factories/factories.py]: ditto

File: backend/apps/volontulo/tests/models/test_factories.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        and has more than 0 char"""
+        # last_add_user = User.objects.last()
+        self.assertTrue(isinstance(self.last_add_user.first_name, str))
+        self.assertTrue(isinstance(self.last_add_user.last_name, str))
+        self.assertTrue(len(self.last_add_user.first_name) > 0)
+        self.assertTrue(len(self.last_add_user.last_name) > 0)
+
+    def test_userfactory_faker_if_email_has_at(self):
+        """test wether email of last created user has an @ """
+        self.assertTrue(self.last_add_user.email.find('@') >= 0)
```
Comment (magul) [inline backend/apps/volontulo/tests/models/test_factories.py]: `self.assertIn('@', self.last_add_user.email)` see: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn

File: backend/apps/volontulo/tests/models/test_factories.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+            self.last_add_user.email,
+            self.last_add_user.username
+            )
+
+
+class OrganizationFactoryTestCase(TestCase):
+    """Test for OrganizationFactory"""
+    def setUp(self):
+        """Set up each test"""
+        # test if organization is created
```
Comment (magul) [inline backend/apps/volontulo/tests/models/test_factories.py]: incorrect comment

File: backend/apps/volontulo/tests/models/test_factories.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+class UserFactoryTestCase(TestCase):
+    """Test for UserFactory"""
+    def setUp(self):
+        u"""setting up each test"""
+        # create user
+        UserFactory.create(first_name="Kornik", last_name="WDrzewie")
+        # create fake user
+        UserFactory.create()
+        self.last_add_user = User.objects.last()
```
Comment (magul) [inline backend/apps/volontulo/tests/models/test_factories.py]: I don't believe, that `last` will return last added object alway (it's not promised), so maybe intead of
```
UserFactory.create()
self.last_add_user = User.objects.last()
```
we can simply do something like that:
```
self.totally_fake_user = UserFactory.create()
```

**Pull request merged**